app/controllers/candidate_interface/sign_up_controller.rb

```diff
@@ -5,13 +5,20 @@ def new
     end
 
     def create
-      @candidate = Candidate.new(candidate_params)
+      @candidate = Candidate.find_by(candidate_params)
 
-      if @candidate.save
-        MagicLinkSignUp.call(candidate: @candidate)
-        render :show
+      if @candidate
+        MagicLinkSignIn.call(candidate: @candidate)
+        render 'candidate_interface/sign_in/show'
```

Comment:
is not this `render :show`

Reply:
The difference is we are in the `sign_up` controller, this is requesting from `sign_in`.

Reply:
See https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/248#discussion_r330536851.